src/enums/generated/ActionGroup.generated.cpp

```diff
@@ -116,7 +116,7 @@ auto ActionGroup_fromString(const string& value) -> ActionGroup {
         return GROUP_ZOOM_FIT;
     }
 
-    g_warning("Invalid enum value for ActionGroup: \"%s\"", value.c_str());
+    g_warning("Invalid enum value for ActionGroup: «%s»", value.c_str());
```

Comment:
This change was made by `generateConvert.php`.

Reply:
It looks like the `generateConvert.php` script should be updated to produce the desired result, see #2407.

Reply:
See #3271